test/AuthTokenVerifier.test.ts

```diff
@@ -117,18 +115,36 @@ describe("AuthTokenVerifier", () => {
       });
     };
 
-    describe("With Ethers signature", () => {
+    describe("With ethers.Signer", () => {
+      beforeAll(() => {
+        delegate = Wallet.createRandom();
+        createClaim = createEIP191claim;
+      });
+      delegateTests();
+    });
+
+    describe("With Keys signer", () => {
       beforeAll(() => {
-        createClaim = createEthersClaim;
+        delegate = new Keys();
+        createClaim = createEIP191claim;
       });
       delegateTests();
     });
 
-    describe("With Json web signature", () => {
+    describe.only("With P256 signer", () => {
```

Comment:
Is this `.only` correct?
```suggestion
    describe("With P256 signer", () => {
```